### ISSUE TITLE
document adherence to Oxford English Dictionary spelling

### ISSUE THREAD

**Issue by michaelficarra** [tc39/ecma262#1649]
I believe it started with ES2015, but at some point we started to follow Oxford English Dictionary spelling conventions. The most noticeable change for me was the renaming of "object initialisers" to "object initializers". @allenwb should know more about this decision. Either way, I think we should document this within the spec, possibly within a new "editorial conventions" section.

**Comment by allenwb** [comment]
I don't have the actual references at hand, but what we discovered back then was that the ISO guidelnes permitted standards to follow the OED conventions and that was it was also acceptable to Ecma. So we adopted those conventions.

**Comment by jmdyck** [comment]
I agree that we should document editorial preferences & decisions, but I'm doubtful that we should do so *within the spec*. Certainly, it makes sense to state an editorial choice in the spec if doing so would help a reader understand the spec. But I don't think that's true of a lot of "style guide" choices.

For things that are less for someone *reading* the spec and more for someone *writing* it (i.e., new proposals. PRs), perhaps we could have a new document in the how-we-work repo.

**Comment by apaprocki** [comment]
I'll just leave this, since it is sufficiently meta -- the British Standards Institution (BSI) publishes a standard, BS 0, the "_standard for standards_", where it literally covers only these 3 items, on page 1, in the _introduction_ (they must come up _really_ often!):

> Certain topics regularly arise during any revision of BS 0, and it was decided:
> • to retain the “z” spelling usage (e.g. “standardization”) for consistency with the preferred variant listed in the Shorter Oxford English Dictionary, noting also that it is the traditional English usage rather than an Americanism;
> • to retain the decimal point in preference to the decimal comma, noting that it reflects the overwhelmingly predominant practice in the UK and is inconsistent with many international and European standards that are adopted without change by BSI; and
> • to retain the “f” spelling of “sulfur” and its derivatives, noting that it is the agreed variant adopted by the International Union of Pure and Applied Chemistry (IUPAC) and most analogous professional bodies in the UK, and reflects the general usage of most of those in the UK scientific community.
